parser.y

```diff
@@ -898,6 +895,9 @@ import (
 	IndexType			"index type"
 	IndexTypeName			"index type name"
 	IndexTypeOpt			"optional index type"
+	IndexPartSpecification		"Index column name or expression"
+	IndexPartSpecificationList	"List of index column name or expression"
+	IndexPartSpecificationListOpt     "Optional list of index column name or expression"
```

Comment:
It's still not formated... Fine, I think we should develop a format tool for *.y.

Reply:
@tangenta Please create an issue 😄